builder/bundleset.go

```diff
@@ -57,6 +57,11 @@ func validateAndFillBundleSet(bundles bundleSet) error {
 				b.AllPackages[k] = v
 			}
 		}
+		for _, optional := range b.OptionalIncludes {
+			for k, v := range bundles[optional].AllPackages {
+				b.AllPackages[k] = v
```

Comment:
Adding optional packages to the all package list for a bundle will insert all of the optional manifest file entries into the original manifest

Reply:
We need to make sure that optionally included manifests do not impact the original manifest while also guaranteeing that mixer will build them.